python/ray/tune/tests/test_searchers.py

```diff
@@ -230,6 +230,26 @@ def testNevergrad(self):
         best_trial = out.best_trial
         self.assertLessEqual(best_trial.config["report"], 2.0)
 
+    def testNevergradWithKwargs(self):
+        from ray.tune.search.nevergrad import NevergradSearch
+        import nevergrad as ng
+
+        np.random.seed(2020)  # At least one nan, inf, -inf and float
+
+        out = tune.run(
+            _invalid_objective,
+            search_alg=NevergradSearch(
```

Comment:
Can we limit the test to just the searcher initialization? Just making sure that initializing `NevergradSearch` with an optimizer that requires args without erroring should be good enough.

Reply:
I totally agree with you. Therefore, for this test case, only verify that nevergrad is instantiated. thanks.